js/views/conversation_view.js

```diff
@@ -1311,6 +1312,17 @@
       }
     },
 
+    async cloakMessages() {
+      var nneElem=document.getElementsByClassName('gutter inactive')[0].style;
```

Comment:
Since the conversation header is a React component, I'd prefer that you implement this such that the react component for the header is re-rendered, but produces nothing. That would be the more React way of doing things. But we can talk about that further when we've agreed on the right user experience.

Also, there's a command in the app - `yarn lint` which I'd recommend you run, because it will ensure that the code matches our expected formatting.

Reply:
I'm not sure what oy mean by the header being re-rendered without producing nothing. The idea to keep the command wheel. I'm not familiar enough with React, but can look it up eventually.